### DISCUSSION TITLE
两套layout.tsx代码是否可以合并呢

### DISCUSSION THREAD

**Discussion by 2010363329** [ant-design/ant-design#46091] [Q&A]
https://ant-design.antgroup.com/docs/react/use-with-next-cn
https://next.nodejs.cn/docs/app/building-your-application/styling/css-in-js
一个是解决页面闪烁的问题 一个是[styled-components]用法的问题 是否可以合并呢 功能会不会冲突

**Comment by MadCcc** [comment]
antd 文档中关于 nextjs 用法就是按照官方推荐来的

**Reply by 2010363329** [reply]
我现在报：Prop `className` did not match. Server: "sc-crrtmM iHYPpp" Client: "sc-bdfCDU dQZUZY"
会不会是因为这个layout.tsx导致的呢 毫无头绪了
